### DISCUSSION TITLE
Released fish for rec

### DISCUSSION THREAD

**Discussion by brianlangseth-NOAA** [pfmc-assessments/canary_2023#41] [General]
@JohnBudrick, I looked at the rec comps for released and retained. You are right in that released fish are smaller (all from the PC). See figure below. 
![rec_lenDensity_retained](https://user-images.githubusercontent.com/27824606/214928770-c6db455c-0b26-4b59-805c-a7b0a3fd6085.png)

After thinking about it I plan to exclude released fish. My rationale is that the comps track the sizes of fish that catches remove from the population. Even though released fish represent the possible sizes the gear "samples", and mortality from them is included in total removals, I plan to keep just retained. This seems to match our conversations around quillback.  This does influence how we approach discards, and vice versa. If we choose to model retention we may want to put these back in, but at the moment I plan to exclude.

Do you have issue with this?

**Comment by JohnBudrick** [comment]
Hi Brian,

The lengths also aren't necessarily sampled in proportion to one another as lengths on discards sampled onboard are from a variable number of anglers compared to those sampled for length of retained catch that are at least known.  Thus you would probably need to weight the comps for the released and retained fish by the estimates of retained for PR and PC vs. discarded dead for PR and PC, then block selectivity between 2003-2016 to due to the smaller fish from including discards then vs before and after, then exclude the discards thereafter and return to the pre 2003 block selectivity or start a new one.  That's a bit of work and parameters.  Conversely, I am concerned about the low sample size from PR retained during that period in the absence of the PC discards with larger fish given the distance they are willing to go for fish vs PR and the sample size effects on representation of larger fish, and thus the perception of the larger fish being absent without a time block.  Perhaps just a sensitivity. Thoughts?

Thanks,

John




From: Brian Langseth ***@***.***>
Sent: Thursday, January 26, 2023 11:23 AM
To: pfmc-assessments/canary_2023 ***@***.***>
Cc: Budrick, ***@***.*** ***@***.***>; Mention ***@***.***>
Subject: [pfmc-assessments/canary_2023] Released fish for rec (Discussion #41)

WARNING: This message is from an external source. Verify the sender and exercise caution when clicking links or opening attachments.


@JohnBudrick<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FJohnBudrick&data=05%7C01%7CJohn.Budrick%40wildlife.ca.gov%7C222a29d6e08b48b950ce08daffd2cb26%7C4b633c25efbf40069f1507442ba7aa0b%7C0%7C0%7C638103578060770921%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=IMBV47TGRfvNlRmCS97ontEli%2FIqGoeczaK%2Bi7hr9pA%3D&reserved=0>, I looked at the rec comps for released and retained. You are right in that released fish are smaller (nearly all from the PC). See figure below.
[rec_lenDensity_retained]<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fuser-images.githubusercontent.com%2F27824606%2F214928770-c6db455c-0b26-4b59-805c-a7b0a3fd6085.png&data=05%7C01%7CJohn.Budrick%40wildlife.ca.gov%7C222a29d6e08b48b950ce08daffd2cb26%7C4b633c25efbf40069f1507442ba7aa0b%7C0%7C0%7C638103578060770921%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=PkC1kbacx4V%2FkYRngQzAvW1umicovkXgd4Za%2FFIi2rg%3D&reserved=0>

After thinking about it I plan to exclude released fish. My rationale is that the comps track the sizes of fish that catches remove from the population. Even though released fish represent the possible sizes the gear "samples", and mortality from them is included in total removals, I plan to keep just retained. This seems to match our conversations around quillback. This does influence how we approach discards, and vice versa. If we choose to model retention we may want to put these back in, but at the moment I plan to exclude.

Do you have issue with this?

-

**Comment by brianlangseth-NOAA** [comment]
To be honest I dont follow. My original intent was to confirm your suspicion that the bit of smaller sized fish in PC was due to released fish. Among retained fish it is really clear PR and PC catch similar lengths and therefore combining modes make sense. Lets discuss over the phone after the workshop, but I dont perceive a strong argument to keep released fish in our comps.

**Comment by JohnBudrick** [comment]
If everything was not intended to be released given prohibition, I think I would be more in agreement.  Released due to preference vs. regulation is different.  In this instance released are representative of all of what was caught in the PC mode.   Seems like valuable data especially given the lack of data during that period from commercial, though the trawl survey helps.  We can talk more later.

**Comment by brianlangseth-NOAA** [comment]
This issue was expanded upon at the pre-assessment data workshop. In that presentation the difference between released and retained fish during the period of no-retention is small. This is the period with a majority of released fish. The conclusion is that including released fish in the comps is expected to have a small effect. Can test this effect via sensitivity.

**Comment by JohnBudrick** [comment]
The discarded fish measured by samplers onboard CPFVs before release during the period of no retention include fish that otherwise would have been retained, so I would only expect a minor shift in the LF distribution to the left from fish that otherwise would have been discarded compared to retained catch from the retained catch when retention was allowed.  Not sure a separate selectivity block during prohibition would be needed for the PC mode to account for the smaller fish sampled onboard CPFVs if they are included for comparison to years when only retained fish lengths were available before 2003.

**Reply by brianlangseth-NOAA** [reply]
We plan on applying a selectivity block before and after the period of no retention, but because we are combining PC and PR modes within a single rec fleet as has been done in the past, it will apply to both sectors.
